pkg/controllers/disruption/controller.go

```diff
@@ -221,32 +220,23 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command,
 	c.cluster.MarkForDeletion(providerIDs...)
 
 	if err := c.queue.Add(orchestration.NewCommand(nodeClaimNames,
-		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, m.Type(), m.ConsolidationType())); err != nil {
+		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, string(m.Reason()), m.ConsolidationType())); err != nil {
```

Comment:
```suggestion
		lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode }), commandID, m.Reason(), m.ConsolidationType())); err != nil {
```

Should you just type the reason in so that you take it in as a `v1.DisruptionReason`?